badge.go

```diff
@@ -12,10 +12,11 @@ import (
 )
 
 type badge struct {
-	Subject string
-	Status  string
-	Color   Color
-	Bounds  bounds
+	Subject    string
+	Status     string
+	LabelColor Color
```

Comment:
I have named the var `labelColor`, which is named similar to other libs 
Example: https://www.npmjs.com/package/badge-maker